### PR TITLE
Add JAVA_HOME setup command for Java9

### PR DESCRIPTION
**Pull request by Nkzn**
Java 9 changes the parameter format for `java_home`.

### CODE REVIEW

File: Java/README.md [hunk 1]
```diff
@@ -21,4 +21,8 @@ Add `JAVA_HOME` to your environment variables by adding the line below to your `
 
     export JAVA_HOME="`/usr/libexec/java_home -v 1.8`"
 
+Are you Java9 user? Use below instead of above.
```
Comment (Kyslik) [inline Java/README.md]: I would rephrase it into:

>In case you are using Java9, use the following:
>    export JAVA_HOME="/usr/libexec/java_home -v 9"

File: Java/README.md [hunk 2]
```diff
@@ -21,4 +21,8 @@ Add `JAVA_HOME` to your environment variables by adding the line below to your `
...
+
+    export JAVA_HOME="`/usr/libexec/java_home -v 9`"
```
Comment (Kyslik) [inline Java/README.md]: Are backticks actually needed?

**Comment by sb2nov** [comment]
@Nkzn ping?

File: Java/README.md (hunk 2 above)
Comment (simeg) [reply Java/README.md]: They are indeed needed, I just tried it. And we have them for the Java 8 example above.

**Comment by simeg** [comment]
I just pushed a change for the sentence, so I think your comments are fixed @Kyslik.

I'll merge this now. I hope it's ok! Thanks @Nkzn.

**Comment by Kyslik** [comment]
Perfect, thanks! :)

**Comment by Nkzn** [comment]
Thanks all! sorry... 🙏
